app/src/main/res/layout/fragment_geraete.xml

```diff
@@ -23,6 +23,29 @@
             tools:context=".ui.geraete.GeraeteFragment">
 
 
+            <Button
+                android:id="@+id/geraete_button_auswertung"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
```

Comment:
Muss dann auch noch in dimens ausgelagert werden

Reply:
Das müssen wir generell noch machen, ist auch als TODO bei Trello eingetragen.